trieste/acquisition/combination.py

```diff
@@ -24,24 +24,20 @@
 
 class Reducer(AcquisitionFunctionBuilder):
     r"""
-    A :class:`Reducer` builds an :func:`~trieste.acquisition.AcquisitionFunction` whose output is
-    calculated from the outputs of a number of other
-    :func:`~trieste.acquisition.AcquisitionFunction`\ s. How these outputs are composed is
-    defined by the method :meth:`_reduce`.
+    A :class:`Reducer` builds an `~trieste.acquisition.AcquisitionFunction` whose output is
+    calculated from the outputs of a number of other `~trieste.acquisition.AcquisitionFunction`\ s.
+    How these outputs are composed is defined by the method :meth:`_reduce`.
     """
 
     def __init__(self, *builders: AcquisitionFunctionBuilder):
         r"""
         :param \*builders: Acquisition function builders. At least one must be provided.
-        :raise ValueError: If no builders are specified.
+        :raise ValueError (or InvalidArgumentError): If no builders are specified.
```

Comment:
can a ValueError be raised by this function?

Reply:
no it appears not. tf.debugging seems to raise InvalidArgumentError except for assert_shapes and other shape-based checks, where it can also raise ValueError. I'll change this to InvalidArgumentError